src/js/media-controller.js

```diff
@@ -1140,7 +1140,19 @@ const getStreamType = (controller) => {
   
   if (!media) return undefined;
   
-  if (media.streamType) return media.streamType;
+  if (media.streamType) {
+    // If the slotted media supports `streamType` but
+    // `streamType` is "unknown", prefer `default-stream-type`
+    // if set (CJP)
+    if (media.streamType === 'unknown' && controller.hasAttribute('default-stream-type')) {
+      const defaultType = controller.getAttribute('default-stream-type');
+
+      if (StreamTypeValues.includes(defaultType)) {
+        return defaultType;
```

Comment:
I was considering if we should have a default default stream type, e.g. if no default stream type then it's on-demand. But I think ultimately that can be up to the player designer to decide how to treat the "no info" default. 👍